pkg/controller/servicebindingrequest/reconciler.go

```diff
@@ -239,8 +250,12 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 
 	// updating status of request instance
 	if err = r.updateStatusServiceBindingRequest(sbr, &sbrStatus); err != nil {
-		log.Error(err, "On updating status of ServiceBindingRequest.")
-		return RequeueError(err)
+		// requeue only in case of conflict
+		if k8serrors.IsConflict(err) {
```

Comment:
@isutton `k8serrors` is not the alias used for that import throughout the code-base. Please use `errors` directly instead.

Reply:
The `errors` package is already being exported and not aliased, and additionally the `k8serrors` package is an alias to `k8s.io/apimachinery/pkg/api/errors` which goal is to provide predicates for several Kubernetes specific errors.

Reply:
Well, in that case, we should add an additional function in `common.go`, in order to handle this type of return, following what's being done for the other Reconciler return statements.

Reply:
Solved in 0a84edcfec0f1086902b4a68d458954f46ab3885.